.vscode/launch.json

```diff
@@ -1,16 +1,39 @@
 {
-	"version": "0.1.0",
-	"configurations": [
-		{
-			"name": "Launch Extension",
-			"type": "extensionHost",
-			"request": "launch",
-			"runtimeExecutable": "${execPath}",
-			"args": ["--extensionDevelopmentPath=${workspaceRoot}/gclient" ],
-			"stopOnEntry": false,
-			"sourceMaps": true,
-			"outFiles": [ "${workspaceRoot}/gclient/out/src/**/*.js" ],
-			"preLaunchTask": "npm"
-		}
-	]
-}
\ No newline at end of file
+  "version": "0.1.0",
+  "configurations": [
+    {
+      "name": "Launch Extension",
+      "type": "extensionHost",
+      "request": "launch",
+      "runtimeExecutable": "${execPath}",
+      "args": ["--extensionDevelopmentPath=${workspaceRoot}/gclient"],
+      "stopOnEntry": false,
+      "sourceMaps": true,
+      "outFiles": ["${workspaceRoot}/gclient/out/src/**/*.js"],
+      "preLaunchTask": "npm"
+    },
+    {
+      "name": "Language Server",
```

Comment:
Maybe this is because of my workflow was different? 

I opened VS code at the root of the repo, so both `gClient` and `gServer` are there, and VS code didn't seem to see the `./gserver/.vscode/launch.json` settings. 

So when I debug, I first *Launch Extension* which opens an instance of VSCode with the local extension, then I attach to that instance's `gserver` with the *Launch Server* debugger (so two debuggers are running).

Reply:
I've updated the code with more clear descriptions of what's happening.